models/issues/review.go

```diff
@@ -474,6 +475,35 @@ func SubmitReview(doer *user_model.User, issue *Issue, reviewType ReviewType, co
 	return review, comm, committer.Commit()
 }
 
+// GetReviewOptions represent filter options for GetReviewByOpts
+type GetReviewOptions struct {
+	IssueID    int64
+	ReviewerID int64
+	Dismissed  util.OptionalBool
+}
+
+// GetReviewByOpts return reviews based on GetReviewOptions
+func GetReviewByOpts(ctx context.Context, opts *GetReviewOptions) ([]*Review, error) {
```

Comment:
Hmm the function name looks odd, we don't currently have any function that is in the form of `GetxxxByOpts`, think it should rather be `GetReviews` or `FindReviews`.

Reply:
I wont mind renaming things ...